float_data_for_tock.py

```diff
@@ -0,0 +1,153 @@
+import requests
+import os
+import json
+from datetime import datetime, timedelta
+
+message = '\nResults of operation:'
+print(message)
+print(len(message) * '=')
+
+"""
+Set up API request.
+"""
+url = 'https://api.floatschedule.com/api/v1/'
+headers = {'Authorization': 'Bearer ' + os.environ.get('FLOAT_API_KEY')}
+
+"""
+Fetch task data  for prior and next six months.
+"""
+endpoint = 'tasks'
+target_date = datetime.today() - timedelta(weeks=26)
+year = target_date.year
+month = target_date.month
+day = target_date.day
+weeks = '52'
+request_date = str(year) + '-' + str(month) + '-' + str(day)
+payload = {'start_day': request_date, 'weeks': weeks}
+r = requests.get(url + endpoint, headers=headers, params=payload)
+task_data = json.loads(r.content.lower().strip())
+print('\nFetched task items from Float.')
+
+"""
+Fetch people data for all users from Float.
+"""
+endpoint = 'people'
+r = requests.get(url + endpoint, headers=headers)
+people_data = json.loads(r.content.lower().strip())
+print('\nFetched people items from Float.')
+
+"""
+Clean people data for all users from Float to remove all but req'd fields.
+"""
+clean_data = list()
```

Comment:
Not that I get to be the arbiter of what's pythonic, but IMO comprehensions are more idiomatic than `map`, and mixing the two is weird. I would write something like:

```python
clean_data = [
    {'im': person['im'], 'people_id': person['people_id']}
    for person in people_data
]
```